crates/oxc_cli/src/lint/mod.rs

```diff
@@ -179,4 +181,10 @@ mod test {
             get_lint_options("lint --ignore-pattern ./test --ignore-pattern bar.js foo.js");
         assert_eq!(options.ignore_pattern, vec![String::from("./test"), String::from("bar.js")]);
     }
+
+    #[test]
+    fn list_rules_true() {
+        let options = get_lint_options("lint foo.js --rules");
```

Comment:
It would be great if this works without the file arg as well

Reply:
The path is required though 🤔

Reply:
I'm not super familiar with `clap` but is it possible to make an arg optional depending on another arg? i.e. `PATH` can be optional if `--rules` is present

Reply:
Seems like we want this: https://docs.rs/clap/4.2.2/clap/struct.Arg.html#method.required_unless_present 👍🏼